scripts/init_host_server

```diff
@@ -35,12 +35,12 @@ mkdir -p /opt/pool/bin
 export POOL_BIN_PATH=/opt/pool/bin
 export PATH=${POOL_BIN_PATH}:$PATH:/usr/local/bin
 
-if [[ -f /etc/os-release ]] && [[ $(cat /etc/os-release | grep "NAME=CoreOS") ]]; then
-    echo "CoreOS detected. Updating to latest docker.."
+if [ `which systemctl` ]; then
```

Comment:
I recommend to use dollar sign instead of backquote. [ref](http://stackoverflow.com/questions/9405478/command-substitution-backticks-or-dollar-sign-paren-enclosed)